packages/api/src/command/medical/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -13,35 +19,38 @@ export type XCUrls = {
 
 export function parseCQDirectoryEntries(orgsInput: Organization[]): CQDirectoryEntryData[] {
   const orgs = orgsInput.flatMap(org => {
-    const orgOid = org?.identifier?.value?.value;
+    if (!org) {
+      return [];
+    }
+    const orgOid = org.identifier?.value?.value;
     if (!orgOid) return [];
 
-    const url = getUrls(org?.contained);
+    const url = getUrls(org.contained);
     if (!url?.urlXCPD) return [];
-    let oid = org?.identifier?.value?.value;
+    const oid = org.identifier?.value?.value;
     try {
-      oid = normalizeOid(org?.identifier?.value?.value);
+      const normalizedOid = normalizeOid(oid);
+      const coordinates = getCoordinates(org.address);
+      const state = getState(org.address);
+
+      const orgData: CQDirectoryEntryData = {
+        oid: normalizedOid,
+        name: org.name?.value ?? undefined,
+        urlXCPD: url.urlXCPD,
+        urlDQ: url.urlDQ,
+        urlDR: url.urlDR,
+        lat: coordinates?.lat ?? undefined,
+        lon: coordinates?.lon ?? undefined,
+        data: {
```

Comment:
No need to deconstruct if its just this object... a simple `data: org` suffices.